droidlet/dashboard/web/src/components/Memory2D/ClusteredObjsPopup.js

```diff
@@ -155,7 +167,7 @@ export function positionClusteredObjsPopup(
 }
 
 function shortenLongTableEntries(e) {
-  if (e.length > 16) {
+  if (e && e.length > 12) {
```

Comment:
It's better to define constants rather than just put numbers in the code - for one thing, it explains to future developers the intent behind these numbers, but also it helps if the number is currently - or will be in the future - used in more than one place in the code.

Reply:
Fair enough, compiled a ton of these numbers into a new `Memory2DConstants` file